### PR TITLE
Adding the new Service Map widget

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?

Adds docs for the new Service Map widget

### Motivation

yay documenting new functionality before it is actually released!

### Preview link

https://docs-staging.datadoghq.com/kaylyn/service-map-widget/graphing/widgets/service_map/

### CODE REVIEW

File: content/en/graphing/widgets/service_map.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+### Configuration
+
+1. Select an [environment][1], primary tag (or `*` if it is set up for your account), and a [service][2].
+2. Enter a title for your graph.
+
+### Results
+
+Nodes are sized relative to one another based on the request. The circumference of a node is heavier and colored green for OK, yellow for Warn, red for Alert, and grey if there's no data.
+
+Services connected to the mapped service are sorted outwards from the middle by requests. The five services with the highest requests are shown with labels by default. Additionally, any service with a monitor in Alert state will always have a label.
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/service_map.md]:
```suggestion
Services connected to the mapped service are sorted outwards from the middle by requests. The five services with the highest requests are shown with labels by default. Additionally, any service with a monitor in alert state always has a label.
```

File: content/en/graphing/widgets/service_map.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    "datacenter:*"
+  ]
+}
+```
+
+| Parameter | Type   | Required | Description                                                       |
+|-----------|--------|----------|-------------------------------------------------------------------|
+| viz       | string | yes      | The type of widget, for the service map widget, use `servicemap`. |
+| service   | string | yes      | The ID of the service you want to map.                            |
+| filters   | object | yes      | Your env and primary tag (or `*` if it is for your account).      |
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/service_map.md]:
```suggestion
| filters   | object | yes      | Your env and primary tag (or `*` if enabled for your account).      |
```

File: content/en/graphing/widgets/service_map.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+{{< img src="graphing/widgets/service_map/test_service_map.png" alt="service map setup" responsive="true" style="width:80%;">}}
+
+## Setup
+
+{{< img src="graphing/widgets/service_map/service_map.png" alt="service map setup" responsive="true" style="width:80%;">}}
+
+### Configuration
+
+1. Select an [environment][1], primary tag (or `*` if it is set up for your account), and a [service][2].
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/service_map.md]:
```suggestion
1. Select an [environment][1], primary tag (or `*` if enabled for your account), and a [service][2].
```

File: content/en/graphing/widgets/service_map.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+further_reading:
+- link: "graphing/dashboards/screenboard/"
+  tag: "Documentation"
+  text: "Screenboard"
+- link: "graphing/graphing_json/"
+  tag: "Documentation"
+  text: "Building Dashboard using JSON"
+---
+
+The service map widget displays a map of a service to all of the services that call it, and all of the services that it calls. The node in the center of the widget represents the mapped service. Services that call the mapped service are shown to the left with arrows from the caller to the service. Services that the service calls are displayed to the right, with arrows in the direction of calls.
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/service_map.md]:
```suggestion
This widget displays a map of a service to all of the services that call it, and all of the services that it calls. The node in the center of the widget represents the mapped service. Services that call the mapped service are shown to the left with arrows from the caller to the service. Services that the service calls are displayed to the right, with arrows in the direction of calls.
```

File: content/en/graphing/widgets/service_map.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+{{< img src="graphing/widgets/service_map/service_map.png" alt="service map setup" responsive="true" style="width:80%;">}}
+
+### Configuration
+
+1. Select an [environment][1], primary tag (or `*` if it is set up for your account), and a [service][2].
+2. Enter a title for your graph.
+
+### Results
+
+Nodes are sized relative to one another based on the request. The circumference of a node is heavier and colored green for OK, yellow for Warn, red for Alert, and grey if there's no data.
```
Comment (ruthnaebeck) [inline content/en/graphing/widgets/service_map.md]:
```suggestion
Nodes are sized relative to one another based on the request. The circumference of a node is heavier and colored green for OK, yellow for warn, red for alert, and grey if there's no data.
```

**Comment by apigirl** [comment]
waiting on GA deploy

**Pull request merged**